testing/arkane/two_parameter_arrhenius_fit/supporting_information.csv

```diff
@@ -0,0 +1,6 @@
+Label,Symmetry Number,Number of optical isomers,Symmetry Group,Rotational constant (cm-1),"Calculated Frequencies (unscaled and prior to projection, cm^-1)",Electronic energy (J/mol),"E0 (electronic energy + ZPE, J/mol)",E0 with atom and bond corrections (J/mol),Atom XYZ coordinates (angstrom),T1 diagnostic,D1 diagnostic

```

Comment:
please remove the chemkin, output, and csv files from the commit

Reply:
ok